gdx/src/com/badlogic/gdx/graphics/g3d/loader/ObjLoader.java

```diff
@@ -462,7 +462,7 @@ private void addTexture (ModelMaterial mat, String texFilename, int usage) {
 		}
 
 		public void reset () {
-			ambientColor = Color.WHITE;
```

Comment:
default libGDX shader doesn't support ambient color in material anyway, so i think default to BLACK is probably better.

Reply:
Good point, I end up changing to null so it will not change the current behavior.